packages/lib/src/validations/VanillaContracts.ts

```diff
@@ -0,0 +1,19 @@
+import Contract from '../artifacts/Contract';
+import ContractAST from '../utils/ContractAST';
+
+let VANILLA_CONTRACTS = '@openzeppelin/contracts/';
+
+export function importsVanillaContracts(contract: Contract, buildArtifacts?: any): string[] | null {
+  const ast = new ContractAST(contract, buildArtifacts, { nodesFilter: ['ContractDefinition'] });
```

Comment:
Do we need `ContractDefinition` nodes for this?

Reply:
Ouch, copy-pasta error.